### ISSUE TITLE
Duplicate post request

### ISSUE THREAD

**Issue by ghost** [go-gitea/gitea#6189]
- Gitea version (or commit ref): 594f591
- Git version: 2.20.1
- Operating system: macOS
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:
```
Start migrating repository "repo"
[Macaron] 2019-02-26 10:44:13: Started POST /repo/migrate for [::1]

... (migrating other repositories)

[Macaron] 2019-02-26 10:49:40: Started POST /repo/migrate for [::1]

...

2019/02/26 10:49:40 [I] [SQL] ROLL BACK
2019/02/26 10:49:40 [...routers/repo/repo.go:146 handleCreateError()] [E] MigratePost: repository already exists [uname: Mirror, name: repo]
2019/02/26 10:49:40 [D] Template: status/500

...

2019/02/26 10:50:10 [I] [SQL] COMMIT
2019/02/26 10:50:10 [T] Repository migrated [1285]: Mirror/repo
[Macaron] 2019-02-26 10:50:10: Completed POST /repo/migrate 302 Found in 5m56.741603459s

```

## Description

Sometimes a strange situation happened to me when I migrating a repository

The post request of migrating will be sent twice and caused the error “repository already exists”.

Does anyone have any ideas?

http://geek.starbean.net/?p=393
This article could be help

**Comment by techknowlogick** [comment]
I'm able to replicate this by clicking the migrate button multiple times, perhaps when buttons such as that are clicked they could then be disabled.

**Comment by ghost** [comment]
> I'm able to replicate this by clicking the migrate button multiple times, perhaps when buttons such as that are clicked they could then be disabled.

This is another problem because it will immediately send the POST request twice and display a 500 error page instead of sending the second POST request after a few minutes.

**Comment by ghost** [comment]
This issue can be closed after #6200 is merged.

**Comment by yasuokav** [comment]
I assume it has been fixed.
